### DISCUSSION TITLE
Need to know some class and layout.

### DISCUSSION THREAD

**Discussion by md-rifatkhan** [AlfaazPlus/QuranApp#66] [General]
Assalamu alaikum.

![Screenshot_2023-02-23-22-03-16-94 1](https://user-images.githubusercontent.com/102645154/220962941-cd128616-3c74-444f-a7b7-4f5cb0f7028a.jpg)

At first, I'm a beginner. Can I know which layout and classes are used on this screen? I want to copy only the class and layout used in this screen to implement in my app if you don't mind.

**Comment by faisalcodes** [comment]
Walaikum Assalam!

This activity is called ActivityReaderIndexPage. 

The following code files are used:
- [ActivityReaderIndexPage](https://github.com/AlfaazPlus/QuranApp/blob/master/app/src/main/java/com/quranapp/android/activities/ActivityReaderIndexPage.java)
- [BaseFragReaderIndex](https://github.com/AlfaazPlus/QuranApp/blob/master/app/src/main/java/com/quranapp/android/frags/readerindex/BaseFragReaderIndex.java)
- [FragReaderIndexChapters](https://github.com/AlfaazPlus/QuranApp/blob/master/app/src/main/java/com/quranapp/android/frags/readerindex/FragReaderIndexChapters.java)
- [FragReaderIndexJuz](https://github.com/AlfaazPlus/QuranApp/blob/master/app/src/main/java/com/quranapp/android/frags/readerindex/FragReaderIndexJuz.java)


The following layout files are used:
- [activity_reader_index_page.xml](https://github.com/AlfaazPlus/QuranApp/blob/master/app/src/main/res/layout/activity_reader_index_page.xml)
- [frag_reader_index.xml](https://github.com/AlfaazPlus/QuranApp/blob/master/app/src/main/res/layout/frag_reader_index.xml)


Kindly comply with the [LICENSE](https://github.com/AlfaazPlus/QuranApp/blob/master/LICENSE).

**Reply by md-rifatkhan** [reply]
thank you, but these classes are also connected with many others.

**Reply by faisalcodes** [reply]
You will have to figure it out yourself how the classes are implemented.

**Reply by md-rifatkhan** [reply]
okay thank you

**Reply by faisalcodes** [reply]
If you know how to code, you may also contribute to this repository :)
